setup.py

```diff
@@ -3,161 +3,23 @@
 from distutils.core import Command
 import versioneer
```

Comment:
This is importing the local versioneer.py version so versioneer isn't a dependency for installation.